src/middlewares/errorMiddleware.js

```diff
@@ -0,0 +1,18 @@
+import { ApiError } from '../exeptions/api.error.js';
+
+export const errorMiddleware = (error, req, res, next) => {
+  if (error instanceof ApiError) {
+    res.status(error.status).send({
+      message: error.message,
+      errors: error.errors,
+    });
+  }
+
+  if (error) {
+    res.statusCode = 500;
+
+    res.send({
+      message: 'Server error',
+    });
+  }
```

Comment:
Consider adding a `return` statement after sending the response in both error handling blocks to prevent further execution of the middleware chain.